database/migrations/2020_02_25_101056_create_ports_lldplocportid_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePortsLldplocportidTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('ports_lldplocportid', function (Blueprint $table) {
+            $table->unsignedInteger('device_id');
+            $table->string('lldpLocPortId');
+            $table->unsignedInteger('port_id')->nullable();
+            $table->unique(['device_id', 'lldpLocPortId']);
+        });
```

Comment:
Could you also store the lldpLocChassisId. VMWare ESXi is (mis)using this field to provide the lldpLocPortId currently. This way, I could extend your code later to allow esxi neighbors to be discovered correctly.

Reply:
I will add that. Don't you just love how vendors follow the standards.
I will also update the find port function to or on either of those fields.